digitalocean/README.md

```diff
@@ -9,7 +9,8 @@ This is an simple provisioner for DigitalOcean [Block Storage](https://www.digit
    Create the secret: `kubectl create -f manifests/digitalocean-secret.yaml`
 2. Deploy the provisioner: `kubectl create -f manifests/digitalocean-provisioner.yaml`
 3. Adjust the `hostPath` in [manifests/digitalocean-flexplugin-deploy.yaml](manifests/digitalocean-flexplugin-deploy.yaml)  
-   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`
+   Deploy the flex plugin "installer": `kubectl create -f manifests/digitalocean-flex-plugin-deploy.yaml`.
```

Comment:
I'm not sure if we should end the point/sentences with period, most of the points in the README doesn't.